src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoUtils.java

```diff
@@ -232,6 +233,42 @@ private static ItemStackAndInteger withdrawFromVanillaInventory(AbstractItemNetw
 
         return null;
     }
+    
+    /**
+     * The method is used to withdraw the target item stack of the slot.
+     *
+     * @param network
+     * @param inventories
+     * @param node
+     *            The input node. It's an input node.
+     * @param target
+     *            The input target. It's a chest or some container block.
+     * @param itemStackAndSlot
+     *            The target item stack and slot we want to withdraw.
```

Comment:
Please use `{@Link name}` to refer to a certain thing in the code.
On the first two comments you also practically repeat the same thing twice